src/components/FormField/component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {
+  Form, Input
+} from 'antd'
+import PropTypes from 'prop-types'
+
+const FormField = ({
```

Comment:
are you sure this component will be able to handle all control types? possibly it would be better to name it something like `InputField` and use it as component for text controls